docs/terra/settings.rst

```diff
@@ -133,3 +133,7 @@ Logging Settings
 .. option:: logging.format_style
 
   The format style, ``%``, ``{``, or ``$`` notation. Default: ``%``
+
+.. option:: logging.server.port
+
+  The port that the logging server will listen on. The default is to use the default logging port 9020. However when launching multiple terra runs in parallel, it may become necessary to prevent port collisions. Setting the port to ``0`` will avoid this issue and allow the OS to select a random port whose value will be accessible via ``terra.settings.loggin.server.port``.
```

Comment:
Does the user care about this port? can it always just be defaulted to 0?

Reply:
I flipped back and forth on defaulting it. One one hand you have "don't have to worry about it" and on the other you have "Firewall can get in the way". Ultimately, the plan is to make this used far less in the near future, so I opted for unchanged behavior over new behavior, until after those changes are applied